doc/changelog.md

```diff
@@ -349,6 +349,9 @@
   the `debug.print` function. The issue was caused by non-null-terminated strings.
   [(#418)](https://github.com/PennyLaneAI/catalyst/pull/418)
 
+* Address the issue of missing calls to device destructors at the end of programs.
+  [(#446)](https://github.com/PennyLaneAI/catalyst/pull/446)
```

Comment:
very minor but I imagine this should be moved to the `v0.5.0-dev` section of the changelog?

Reply:
Great catch Josh!